flang/test/Lower/intrinsic-procedures/TRANSER.f90

```diff
@@ -0,0 +1,21 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
This is the same transfer intrinsic as the next one.

Reply:
Yeah, there is a misspelled label in [intrinsic-procedures.f90](https://github.com/flang-compiler/f18-llvm-project/blob/fir-dev/flang/test/Lower/intrinsic-procedures.f90#L1321) right now: `TRANSER`. My original script extracted `TRANSFER` and `TRANSER` as 2 separate intrinsics, sorry! Thanks for pointing this out!